lib/statsd.rb

```diff
@@ -40,9 +45,17 @@ def send(msg)
   #characters that will be replaced with _ in stat names
   RESERVED_CHARS_REGEX = /[\:\|\@]/
 
+  COLON_COLON_REGEX = /::/
```

Comment:
Regexp literals are allocated once by the parser and just emit the `putobject` instruction by the compiler, so it's ok to leave this regexp literal inline in the `send` method